charts/fluentd-elasticsearch/values.yaml

```diff
@@ -63,6 +63,9 @@ elasticsearch:
     enabled: false
     user: null
     password: null
+    passwordSecretKeyRef: null
```

Comment:
Imho this should look like:

```
    existingSecret:
      name: ""
      key: ""
```

In the deamonsert you can than just check for:

```
{{- if .Values.elasticsearch.auth.existingSecret.name }}
```

Reply:
Suggestion applied